Parse/TPTP.cpp

```diff
@@ -2775,7 +3328,7 @@ void TPTP::endFormula()
   case FORALL:
   case EXISTS:
     f = _formulas.pop();
-    _formulas.push(new QuantifiedFormula((Connective)con,_varLists.pop(),_sortLists.pop(),f));
+    _formulas.push(new QuantifiedFormula((Connective)con,_varLists.pop(),_sortLists.pop(),f)); //TODO this is broken currently.
     _states.push(END_FORMULA);
```

Comment:
Sounds scary. Could you explain more? Is it just a weird case that is broken?

Reply:
That is a very old comment that I unfortunately left lying around. It all appears to work at the moment.

Reply:
Great!